src/components/DropzoneAreaBase.md

```diff
@@ -10,6 +10,7 @@ import { DropzoneAreaBase } from 'material-ui-dropzone';
 <DropzoneAreaBase
   onAdd={(fileObjs) => console.log('Added Files:', fileObjs)}
   onDelete={(fileObj) => console.log('Removed File:', fileObj)}
+  onAlert={(message, variant) => console.log(`${variant}: ${message}`)}
```

Comment:
Hmm, I would argue this is not basic usage, however until we have something like story book this is probably the best way to test it out. Perhaps when we have storybook we can have one example with all the customisation in

Reply:
Agree, simply lack of a better place.